rpki-validator/src/main/java/net/ripe/rpki/validator3/storage/data/TrustAnchor.java

```diff
@@ -98,4 +101,11 @@ public X509ResourceCertificate getCertificate() {
     public void markInitialCertificateTreeValidationRunCompleted() {
         this.initialCertificateTreeValidationRunCompleted = true;
     }
+
+    public List<URI> getLocationsByPreference() {
+        return locations.stream()
+            .sorted(Comparator.naturalOrder())
```

Comment:
Just to confirm the intent, sorting in the ends put https in front of rysnc, right?

Reply:
Sorting would re-order urls with the same scheme alphabetically. I changed this.